src/letters/LetterPoints.js

```diff
@@ -0,0 +1,30 @@
+const letterPoints = [ 
+    { key: 1, letter: 'A', point: 1 }, 
```

Comment:
What's the `key` field used for here? It seems like it could just be derived from the index